Competition2018/src/main/java/competition/commandgroups/AutoPutCubeOnScaleCommandGroup.java

```diff
@@ -27,9 +27,9 @@ public AutoPutCubeOnScaleCommandGroup(
         driveToDistance.setDeltaDistance(243);
         wristSet.setGoalAngle(0);
         
-        this.addParallel(elevatorSet);
+        this.addParallel(elevatorSet, 1);
```

Comment:
now that set position commands run forever, they need to timeout in autonomous groups.